src/components/TemplateNavLink.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+type Props = {
+  to: string,
+  textToDisplay: string,
+};
+
+export const TemplateNavLink: FC<Props> = ({ to, textToDisplay }) => (
+  <NavLink
+    to={to}
+    className={({ isActive }) => cn('navbar-item', { 'is-active': isActive })}
+  >
+    {textToDisplay}
+  </NavLink>
+);
```

Comment:
use React memo

Reply:
Thanks! I guess I would use it for the entire Navbar then.